### ISSUE TITLE
TypeError: Cannot read property 'replace' of undefined at Module.extractRequires [as _extractor]

### ISSUE THREAD

**Issue by crellison** [facebook/react-native#11490]
### Description

I recently upgraded my react-native package and some others in my package.json to the latest release. Each time I try to compile me project, I run into a this error in the terminal:

```
TypeError: Cannot read property 'replace' of undefined
    at Module.extractRequires [as _extractor] (PATH_TO_PROJECT/node_modules/react-native/packager/react-packager/src/node-haste/lib/extractRequires.js:31:10)
    at codePromise.then.result (PATH_TO_PROJECT/node_modules/react-native/packager/react-packager/src/node-haste/Module.js:209:49)
    at tryCallOne (PATH_TO_PROJECT/node_modules/promise/lib/core.js:37:12)
    at PATH_TO_PROJECT/node_modules/promise/lib/core.js:123:15
    at flush (PATH_TO_PROJECT/node_modules/asap/raw.js:50:29)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

The simulator throws the following error:

```
{"type":"InternalError","message":"react-packager has encountered an internal error, please check your terminal error output for more details"}
```

I can't tell if this is a source code problem, or if I messed up somehow in updating my npm packages.

### Additional Information

* React Native version: 0.39.2
* Platform: iOS
* Operating System: MacOS

**Comment by lacker** [comment]
How did you upgrade? I suggest using http://facebook.github.io/react-native/blog/2016/12/05/easier-upgrades.html to do upgrades.

**Comment by crellison** [comment]
I managed to sort this out. Thanks for the link though.

**Comment by shkfnly** [comment]
@crellison How did you sort this out?

**Comment by crellison** [comment]
@shkfnly I wish I could tell you. I was updating my RN package and some other RN npm modules at the same time and lost track of what was new and what was old. I ended up updating everything to the latest release regardless and looked at recently posted issues on each page. This one may have been due to a mismatch of ReactNative and React, but I'm really not sure at this point. I wish I could help more...
